.github/CODEOWNERS

```diff
@@ -38,24 +38,24 @@ go.sum @ajm188 @deepthi @harshit-gangal @mattlord @rohit-nayak-ps @systay @froui
 /go/vt/discovery/*tablet_picker* @rohit-nayak-ps @mattlord
 /go/vt/mysqlctl @deepthi @mattlord
 /go/vt/proto @deepthi @harshit-gangal @mattlord
-/go/vt/proto/vtadmin @ajm188 @notfelineit
+/go/vt/proto/vtadmin @notfelineit
 /go/vt/schema @mattlord @shlomi-noach
-/go/vt/servenv @deepthi @ajm188
+/go/vt/servenv @deepthi @dbussink
 /go/vt/schemadiff @shlomi-noach @mattlord
 /go/vt/sqlparser @harshit-gangal @systay @GuptaManan100
 /go/vt/srvtopo @deepthi @mattlord
 /go/vt/sysvars @harshit-gangal @systay
 /go/vt/topo @deepthi @mattlord
 /go/vt/topotools @deepthi @mattlord
 /go/vt/vitessdriver @harshit-gangal
-/go/vt/vtadmin @ajm188 @notfelineit @rohit-nayak-ps
-/go/vt/vtctl @ajm188 @deepthi @rohit-nayak-ps
+/go/vt/vtadmin @notfelineit @rohit-nayak-ps
+/go/vt/vtctl @deepthi @rohit-nayak-ps
 /go/vt/vtctl/vtctl.go @notfelineit @rohit-nayak-ps
-/go/vt/vtctl/grpcvtctldclient @ajm188 @notfelineit @mattlord
-/go/vt/vtctl/grpcvtctldserver @ajm188 @notfelineit @mattlord
-/go/vt/vtctl/reparentutil @ajm188 @GuptaManan100 @deepthi
-/go/vt/vtctl/vtctldclient @ajm188 @notfelineit @mattlord
-/go/vt/vtctld @ajm188 @deepthi @notfelineit @rohit-nayak-ps @mattlord
+/go/vt/vtctl/grpcvtctldclient @notfelineit @mattlord
+/go/vt/vtctl/grpcvtctldserver @notfelineit @mattlord
+/go/vt/vtctl/reparentutil @GuptaManan100 @deepthi
+/go/vt/vtctl/vtctldclient @notfelineit @mattlord
+/go/vt/vtctld @deepthi @notfelineit @rohit-nayak-ps @mattlord
```

Comment:
@rohit-nayak-ps I wasn't sure whether I should add you to vtctl and vtadmin in this file. Please feel free to push a commit.